iso20022-ch-impl/src/test/java/ch/dvbern/oss/lib/iso20022/pain008/v00102ch03/Pain008ServiceTest.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2017 DV Bern AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * limitations under the License.
+ */
+
+package ch.dvbern.oss.lib.iso20022.pain008.v00102ch03;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.xmlunit.builder.DiffBuilder;
+import org.xmlunit.diff.Diff;
+
+import static org.junit.Assert.assertFalse;
+
+/**
+ * Tests for the {@link Pain008Service}.<br>
+ *
+ * The resulting XML will be saved to STORE_PATH if WRITE_TO_FILE is set to true.
+ */
+public class Pain008ServiceTest {
+
+	private static final boolean WRITE_TO_FILE = true;
+	private static final String STORE_PATH = "target/pain008Reference.xml";
+
+	private Pain008V00102CH03Service pain008Service = new Pain008V00102CH03Service();
+
+	private Pain008DTO painDto;
```

Comment:
What is the purpose of this field? Why not simply refactor the init() method to return a Pain008DTO?

Reply:
done